Polyhedron/demo/Polyhedron/Plugins/IO/Selection_io_plugin.cpp

```diff
@@ -10,7 +10,12 @@ class Polyhedron_demo_selection_io_plugin :
     Q_INTERFACES(CGAL::Three::Polyhedron_demo_io_plugin_interface)
     Q_PLUGIN_METADATA(IID "com.geometryfactory.PolyhedronDemo.PluginInterface/1.0")
 public:
+#ifdef USE_SURFACE_MESH
+    QString name() const { return "selection_io_sm_plugin"; }
+#else
```

Comment:
Is this suited? (it looks that the ifdef was belonging to a preliminary version)